.github/workflows/checks.yml

```diff
@@ -19,15 +19,28 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
-      - uses: actions/setup-java@v3
+
+#        # TODO: (#310) Remove when migrating to macOS arm64 runner
+#      - name: Clone Cairo repos (macOS only)
+#        if: matrix.os == 'macos-latest'
+#        run: |
+#          git clone https://github.com/starkware-libs/cairo.git cairo1
+#          git clone https://github.com/starkware-libs/cairo.git cairo2
+#          cd cairo1 && git checkout v1.1.1 && cd ..
+#          cd cairo2 && git checkout v2.2.0 && cd ..
```

Comment:
It was discussed offline, but I since then I got a strong feeling we shouldn't be making things harder for new users who might be running macOS x64 😅